### DISCUSSION TITLE
Is there a dockerized version?

### DISCUSSION THREAD

**Discussion by fjrsaracho** [kalai-transpiler/kalai#66] [Q&A]
Like docker run kalai-transpiler -v ./in:/opt/in -v ./out:/opt/out

**Comment by echeran** [comment]
We don't have a dockerized version at the moment, but that would be nice to have. We have [some notes](https://github.com/kalai-transpiler/kalai/blob/main/docs/Contributing.md#local-system-config) on how to build, for the time being.
